keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1234,8 +1258,22 @@ type DeviceAttribute struct {
 
 // DeviceAttributeMaxValueLength is the maximum length of a string or version attribute value.
 const DeviceAttributeMaxValueLength = 64
+
+// DeviceCapacity describes a quantity associated with a device.
+type DeviceCapacity struct {
```

Comment:
I'm going to OK this, since I am late to the conversation, but I will be looking to understand this change more :)